packages/node-core/src/indexer/sandbox.ts

```diff
@@ -46,11 +51,65 @@ export class Sandbox extends NodeVM {
         },
       })
     );
+    this.root = option.root;
+
+    const filename = 'dist/index.js';
```

Comment:
I don't think this will work with published packages. Can we test from IPFS please